styles/base/_links.scss

```diff
@@ -58,13 +57,32 @@
   &:hover,
   &:active,
   &:focus {
-    transition: $sprk-link-transition;
     color: $sprk-link-light-hover-color;
     font-weight: $sprk-link-light-hover-font-weight;
     border-bottom: $sprk-link-light-hover-border-bottom;
   }
 }
 
+/// Used when an inline link is on the dark background of the footer.
+.sprk-b-Link--footer {
```

Comment:
This class name is very similar to the existing `sprk-c-Footer__link`. Is the difference that the new class is specifically for inline use? What about `sprk-c-Footer__link--inline`? Or `sprk-b-Link--dark-background`?

Reply:
Here are some considerations:
- There is a class called `sprk-b-Link sprk-b-Link--light` that's the link soley used in the footer
- I considered making a `sprk-c-Footer__link--inline` but then I would have to recreate all link styles and interactions there
- I think we're getting closer with `sprk-b-Link--dark-background` but we also have a `--light` version that's confusing.
- This isn't actually being documented anywhere

🧃What about renaming it to `sprk-b-Link--inline-light`

Reply:
I like `sprk-b-Link--inline-light`

Reply:
done! good convo

Reply:
done!